src/main/java/net/atomique/ksar/UI/AboutBox.java

```diff
@@ -84,17 +77,28 @@ private void initComponents() {
 
         jPanel2.setLayout(new javax.swing.BoxLayout(jPanel2, javax.swing.BoxLayout.PAGE_AXIS));
 
-        urllabel.setText("website: http://sourceforge.net/projects/ksar/");
+        String url;
+
+        url ="https://github.com/vlsi/ksar";
+        urllabel.setText(String.format("<html> Website :   <a href=\"\">%s</a></html>", url ));
+        urllabel.setCursor(new Cursor(Cursor.HAND_CURSOR));
         jPanel2.add(urllabel);
+        goWebsite(urllabel, url);
 
-        authorlabel.setText("Author: xavier cherif");
+        url ="https://github.com/vlsi/ksar/CONTRIBUTION.md";
```

Comment:
Afaik, it should be `CONTRIBUTING.md` for proper GitHub integration: https://github.com/blog/1184-contributing-guidelines